README.md

```diff
@@ -6,15 +6,797 @@
 [![License](https://img.shields.io/github/license/massdriver-cloud/airlock)](https://github.com/massdriver-cloud/airlock/blob/master/LICENSE)
 
 ## Overview
-Generate JSON Schema from various sources (OpenTofu, Helm)
+
+Translate between JSON Schema and common IaC languages (opentofu, helm, bicep)
 
 ## Getting Started
 
 ### Prerequisites
-- Go version 1.21
+
+- Go version 1.22
 
 ### Installation
+
 To install this package, simply run:
 
 ```bash
 go get -u github.com/massdriver-cloud/airlock
+```
+
+### Usage
+
+#### OpenTofu
+
+OpenTofu -> JSON Schema:
+
+```bash
+airlock opentofu input /path/to/module
+```
+
+<details>
+    <summary>Example</summary>
+
+`main.tf`:
+
+```terraform
+provider "aws" {
+  region = var.region
+}
+
+variable "bucket_name" {
+  description = "The name of the S3 bucket."
+  type        = string
+}
+
+variable "region" {
+  description = "The AWS region to create the S3 bucket in."
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "enable_versioning" {
+  description = "Enable versioning on the S3 bucket."
+  type        = bool
+  default     = false
+}
+
+variable "acl" {
+  description = "The access control list for the S3 bucket."
+  type        = string
+  default     = "private"
+}
+
+resource "aws_s3_bucket" "example" {
+  bucket = var.bucket_name
+  acl    = var.acl
+
+  versioning {
+    enabled = var.enable_versioning
+  }
+
+  lifecycle_rule {
+    id      = "delete-old-versions"
+    enabled = true
+
+    noncurrent_version_expiration {
+      days = 30
+    }
+  }
+
+  tags = {
+    Name        = var.bucket_name
+    Environment = "Dev"
+  }
+}
+
+output "bucket_id" {
+  value = aws_s3_bucket.example.id
+}
+
+output "bucket_arn" {
+  value = aws_s3_bucket.example.arn
+}
+```
+
+JSON output:
+
+```json
+{
+  "properties": {
+    "bucket_name": {
+      "type": "string",
+      "title": "bucket_name",
+      "description": "The name of the S3 bucket."
+    },
+    "region": {
+      "type": "string",
+      "title": "region",
+      "description": "The AWS region to create the S3 bucket in.",
+      "default": "us-east-1"
+    },
+    "enable_versioning": {
+      "type": "boolean",
+      "title": "enable_versioning",
+      "description": "Enable versioning on the S3 bucket.",
+      "default": false
+    },
+    "acl": {
+      "type": "string",
+      "title": "acl",
+      "description": "The access control list for the S3 bucket.",
+      "default": "private"
+    }
+  },
+  "required": [
+    "acl",
+    "bucket_name",
+    "enable_versioning",
+    "region"
+  ]
+}
+```
+
+</details>
+
+JSON Schema -> OpenTofu:
+
+```bash
+airlock opentofu output /path/to/schema.json
+```
+
+<details>
+    <summary>Example</summary>
+
+`schema.json`:
+
+```json
+{
```

Comment:
I don't think this example is accurate. I don't see any mention of `form`, and my airlock instance is erroring.

Reply:
Fixed